src/test/java/com/github/krzysiekjodlowski/ox/ConsoleUITest.java

```diff
@@ -20,21 +21,20 @@
                 , {"1", NumberRange.of(0, 2)}
         };
     }
-
-    @DataProvider
-    public static Object[][] arbitraryNumbersInputNotInTheRange() {
-        return new Object[][]{
-                {"1", NumberRange.of(-1, 0)}
-                , {"1", NumberRange.of(0, 0)}
-                , {"1", NumberRange.of(2, 2)}
-                , {"1", NumberRange.of(2, 3)}
-        };
-    }
+//
+//    @DataProvider
+//    public static Object[][] arbitraryNumbersInputNotInTheRange() {
+//        return new Object[][]{
+//                {"1", NumberRange.of(-1, 0)}
+//                , {"1", NumberRange.of(0, 0)}
+//                , {"1", NumberRange.of(2, 2)}
+//                , {"1", NumberRange.of(2, 3)}
+//        };
+//    }
```

Comment:
Remember to uncomment or delete. You can create issue for this.

Reply:
Deleted for now.